docs/_ext/towncrier_draft_ext.py

```diff
@@ -106,6 +106,21 @@ def _get_draft_version_fallback(strategy: str, sphinx_config: Dict[str, Any]):
     return ' '.join(msg_chunks)
 
 
+def _nodes_from_rst(state: statemachine.State, rst: str) -> nodes.Element:
```

Comment:
Doesn't it return `List[nodes.Node]`? `node.children` is a list
```suggestion
def _nodes_from_rst(state: statemachine.State, rst: str) -> List[nodes.Node]:
```

Reply:
Ah yes you're right !